### PR TITLE
overview: warn about install-config.yaml deletion

### PR DESCRIPTION
**Pull request by dankenigsberg**
I don't understand why it makes sense and find it surprising. Let's warn other readers.

@crawford can you confirm that this is the intended behavior, and maybe explain the logic behind it?

### CODE REVIEW

File: docs/user/overview.md [hunk 1]
```diff
@@ -60,15 +60,15 @@ mv initial/install-config.yaml .
 rm -rf initial
 ```
 
-You can use the saved install-config for future clusters by copying it that install config into the asset directory and then invoking the installer:
+You can use the saved install-config for future clusters by copying it i nto the asset directory and then invoking the installer:
```
Comment (wking) [inline docs/user/overview.md]: "i nto" -> "into"

File: docs/user/overview.md [hunk 2]
```diff
@@ -60,15 +60,15 @@ mv initial/install-config.yaml .
...
 
 ```sh
 mkdir cluster-0
 cp install-config.yaml cluster-0/
 openshift-install --dir=cluster-0 create cluster
 ```
 
-Supplying a previously-generated install-config like this is [explicitly part of the stable API](versioning.md).
+Supplying a previously-generated install-config like this is [explicitly part of the stable API](versioning.md). Do note that the copied install-config would be deleted from the asset directory by the installer.
```
Comment (wking) [inline docs/user/overview.md]: "Do note" -> "note".

**Comment by wking** [comment]
Motivation for the removal and further discussion in #898.

File: docs/user/overview.md (hunk 2 above)
Comment (crawford) [reply docs/user/overview.md]: I'd prefer to use the term "consume". The information isn't lost, it's just been moved.

File: docs/user/overview.md (hunk 1 above)
Comment (dankenigsberg) [reply docs/user/overview.md]: argh, editing docs over a phone is dangerous.

File: docs/user/overview.md [hunk 3]
```diff
@@ -60,7 +60,7 @@ mv initial/install-config.yaml .
 rm -rf initial
```
Comment (dankenigsberg) [inline docs/user/overview.md]: @crawford @wking what is the name of the "hidden file" mentioned above? It is quite mysterious to me.

File: docs/user/overview.md (hunk 2 above)
Comment (wking) [reply docs/user/overview.md]: I think "[at any point before running delete cluster][1]" is an important point to make.  We don't want to give a false sense of security.

[1]: https://github.com/openshift/installer/pull/1122#discussion_r250426690

File: docs/user/overview.md (hunk 3 above)
Comment (wking) [reply docs/user/overview.md]: `.openshift_install_state.json`

**Comment by dankenigsberg** [comment]
> Can you also squash down to one commit?

Does GitHub GUI allow this basic functionality? I'll use the command line if not.

**Comment by wking** [comment]
> > Can you also squash down to one commit?
>
> Does GitHub GUI allow this basic functionality?

Not to my knowledge.

**Comment by wking** [comment]
Aha, looks like you've allowed maintainer edits, because GitHub let me force-push a squash to your branch with 2b2a6a0 -> 4c6fca4.  Does 4c6fca4 look good to you, @dankenigsberg ?

**Pull request merged**